### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by jonathonherbert**
## What does this change?

Bumps a few dependencies across the board to improve our [Snyk scores](https://app.snyk.io/org/guardian/project/8ecf100e-b5f7-43e5-bbaa-c28fcd28eb7a) for this project.

## Dev notes

I've consolidated our application setup across the checker and manager apps. Seems fairly straightforward, but perhaps there's a more idiomatic way to do this?

The bump to LanguageTool has been tested with `./script/js/compare-rule-xml.js`, to ensure we aren't changing checker behaviour. All of the currently active rules (`COMMA_PARENTHESIS_WHITESPACE, DOUBLE_PUNCTUATION, UPPERCASE_SENTENCE_START, WHITESPACE_RULE, SENTENCE_WHITESPACE, PUNCTUATION_PARAGRAPH_END`) are not defined in XML, and are fairly self explanatory – I've tested them all in sandbox locally and they appear to work as expected.

## How to test

It should compile and run as usual. Snyk should be happy(er).

Tested in CODE. One rule needed to be amended (it had an invalid "skip" property that wasn't being respected anyhow) – I've amended the PROD spreadsheet, so there should be no problem deploying to PROD.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -93,7 +93,9 @@ val checker = (project in file(s"$appsFolder/checker"))
       "com.gu" %% "content-api-client-aws" % "0.7",
       "com.gu" %% "content-api-client-default" % capiClientVersion,
       "edu.stanford.nlp" % "stanford-corenlp" % "3.4",
```
Comment (andrew-nowak) [inline build.sbt:95]: do you need to exclude xom here too?

Comment (andrew-nowak) [reply build.sbt:95]: I still see xalan in the dependency tree 
image

Comment (jonathonherbert) [reply build.sbt]: Thanks, yes. [985ffd2](https://github.com/guardian/typerighter/pull/213/commits/985ffd2ede4164712550a250b19f4910eb7f0945)

**Comment by andrew-nowak** [comment]
I get two red checks on the sentence `Hi,this is a test  sentence.` (no space after comma, two spaces between test and sentence) in PROD, but no matches in CODE. Is this a regression?

**Comment by jonathonherbert** [comment]
Thanks for testing! The [CODE](https://docs.google.com/spreadsheets/d/1rnV5HgCdrAOg1Zynl9m5Zu5PRfyUtxkA7-ZOKQdfkSc/edit#gid=88751783) and [PROD](https://docs.google.com/spreadsheets/d/1bLCD7CrGMDAuuk76e0MTF1-DUNgmXrvTc1d5n2UMZR8/edit?resourcekey#gid=1482142974) sheets were not in sync, as we've been doing some testing in CODE – I've just copied the CODE LT rules across, and they should now match PROD, so in CODE you should now see this rule:

Screenshot 2022-12-08 at 14 04 34

**Pull request merged**